src/cow-react/modules/swap/pure/Row/RowDeadline/index.tsx

```diff
@@ -0,0 +1,80 @@
+import styled from 'styled-components/macro'
+import { Trans } from '@lingui/macro'
+
+import { RowFixed } from 'components/Row'
+import { MouseoverTooltipContent } from 'components/Tooltip'
+import { INPUT_OUTPUT_EXPLANATION } from 'constants/index'
+import { StyledInfo } from '@cow/pages/Swap/styleds'
+import { RowSlippageProps } from '@cow/modules/swap/containers/Row/RowSlippage'
+import { StyledRowBetween, TextWrapper } from '@cow/modules/swap/pure/Row/styled'
+import { RowStyleProps } from '@cow/modules/swap/pure/Row/typings'
+import { ThemedText } from 'theme/index'
+import { DEADLINE_LOWER_THRESHOLD_SECONDS } from '@cow/modules/swap/state/EthFlow/updaters'
+
+const ClickableText = styled.button<{ isWarn?: boolean }>`
+  background: none;
+  border: none;
+  outline: none;
+  padding: 0;
+  margin: 0;
+  font-weight: 500;
+  font-size: 13px;
+  color: ${({ isWarn, theme }) => theme[isWarn ? 'text2' : 'text1']};
+`
+
+export const getNativeOrderDeadlineTooltip = (symbols: (string | undefined)[] | undefined) => (
+  <Trans>
+    <p>
+      {symbols?.[0] || 'Native currency (e.g ETH)'} orders require a minimum transaction deadline threshold of{' '}
```

Comment:
Why is this a list?
AFAICT this is the only place where this is used, and if this is only for native, there will always be only 1 symbol at most